### DISCUSSION TITLE
Does docsify support permanent link preview?

### DISCUSSION THREAD

**Discussion by linrongbin16** [docsifyjs/docsify#2339] [Q&A] [answered]
hi, github markdown support preview permanent link:

![Screenshot_2023-12-16-09-32-49-903_com android chrome-edit](https://github.com/docsifyjs/docsify/assets/6496887/e4d1009a-6efb-40d4-afca-1422e2edb967)

but docsify seems doesn't.

![Screenshot_2023-12-16-09-32-24-260_com android chrome-edit](https://github.com/docsifyjs/docsify/assets/6496887/ce1b1d2a-44e1-478d-baa6-aafef7159215)

is there a way to do it?

**Comment by sy-records** [comment, marked as answer]
Duplicate of #2340
